content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Migrating from Dynamic DNS Service to DDNS Updater App"
+description: "This article provides instructions for users migrating from the SCALE Dynamic DNS service to the new ddns-updater application." 
+weight: 15
+aliases:
+tags:
+- scaleddns
+---
+
+
+{{< include file="content/_includes/SCALEServiceDeprecationNotice.md" type="page" >}}
+
+This article provides instructions on how to move from using the deprecated dynamic DNS service to the new **ddns-updater** application. 
+
+The ddns-updater application is a lightweight universal dynanmic DNS (DDNS) updater with web UI. 
+When installed, a container launches with root privileges in order to apply the correct permissions to the ddns-updater directories. 
+Afterwards, the containter runs as a non-root user.
+
+## Before You Begin
+Before you configure the new ddns-updater application:
+
+* Disable the Dynamic DNS service.
+  Go to **System Settings > Services** and disable the service and clear the **Start Automatically** checkbox. 
+  This prevents the service from re-enabling after a system restart.
+
+* Review your Dynamic DNS service settings and note all provider, domain, IP address, port number, URL and credential (username and password) settings.
+
+If establishing a new provider, create the user account before proceeding. Otherwise, use the existing provider details.
+
+If you want to grant access to a specific user (and group) other than using the defaults, add the new non-root administrative user and take note of the UID and GID for this user.
+
+## Migrating from TrueNAS Dynamic DNS Service
+
+After disabling the dynamamic DNS service, install the ddns-updater application. Go to **Apps** click on **Available Applications** and locate the **ddns-updater** application widget.
+
+{{< trueimage src="/images/SCALE/22.12/ddnsUpdaterAppWidget.png" alt="ddns-updater Application Widget" id="1 ddns-updater Application Widget" >}}
+
+Click **Install** to open the **ddns-updater** configuration wizard.
+
+Select the timezone that applies to your server location from the **Timezone** dropdown list.
+
+Click **Add** to the right of **DNS Provider Configuration** to display provider setting options. 
+Select the DDNS provider from the **Provider** dropdown list. 
+Each provider displays the settings required to establish a connection with and authenticate to that provider. 
```

Comment:
Would this be clearer with something like:

Once a provider is selected, the **DNS Provider Configuration** box will display the settings fields required to establish a connection with and authenticate to that provider.

Reply:
never use the word will in a procedure document. It either does or does not do something. Keep to present tense active voice. :-)